comp_sci_gender_bias/getters/dfe_combined_school_data.py

```diff
@@ -0,0 +1,18 @@
+from comp_sci_gender_bias.pipeline.additional_school_info.combine_dfe_school_data import (
+    save_combined_dfe_data,
+)
+from comp_sci_gender_bias import PROJECT_DIR
+import pandas as pd
+
+COMBINED_DFE_PATH = PROJECT_DIR / "inputs/data/dfe_school_info/dfe_combined_dataset.csv"
```

Comment:
Minor for consistency

```suggestion
DFE_COMBINED_PATH = PROJECT_DIR / "inputs/data/dfe_school_info/dfe_combined_dataset.csv"
```